app/facades/foods_facade.rb

```diff
@@ -0,0 +1,18 @@
+class FoodsFacade
+  def self.service
+    @_service ||= FoodsService.new
+  end
+
+  def self.search_by_keyword(word)
+    filtered_results = []
+    
+    service.search_by_keyword(word)[:foods].each do |food|
+      if food[:ingredients].downcase.include?(word)
```

Comment:
Nice filter as an extension! An extra challenge would be to find a solution that can do the filter and map the JSON items into POROs in just *one* enumerable rather than iterating through the whole list twice (which can be a time-consuming operation)